.github/workflows/build-and-publish.yml

```diff
@@ -10,6 +10,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@main
+    - name: Install dependencies
+      run: npm install
+    - uses: ./
+    - name: Run tests
+      run: npm run test
```

Comment:
will whole action fail if tests fail?

Reply:
This is my understanding

Reply:
we should test that once we merge this, or on some test repo if we want to test it before

Reply:
I can't find anything that definitively states this but I think the default behavior of actions is to exit on failure

Reply:
Yes, actions fail if a script returns something other than 0 or throws.